app/config/config.py

```diff
@@ -0,0 +1,86 @@
+KNIGHTS = {
+    "lancelot": {
+        "name": "Lancelot",
+        "power": 35,
+        "hp": 100,
+        "armour": [],
+        "weapon": {
+            "name": "Metal Sword",
+            "power": 50,
```

Comment:
Remove the trailing comma after the 'power' key in the weapon dictionary to ensure proper JSON formatting.